### PR TITLE
Feat: production dockerfile

### PR DESCRIPTION
**Pull request by JMorinPS**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Added legacy Dockerfile from CICD repository
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no

### CODE REVIEW

File: Dockerfile.prod [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+RUN /usr/local/bin/composer self-update --2.2
+
+RUN a2enmod rewrite
+RUN a2dissite 000-default
+
+COPY docker/apache2/prestonbot.conf /tmp/prestonbot.conf.template
+
+RUN export template="$(cat /tmp/prestonbot.conf.template)"; eval "echo \"${template}\"" >/etc/apache2/sites-enabled/prestonbot.conf
+
+#COPY prestonbot/ /var/www/html/
```
Comment (djodjo3) [inline Dockerfile.prod:41]: Still necessary ? 😇

File: Dockerfile.prod [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ENV SYMFONY_PHPUNIT_VERSION $SYMFONY_PHPUNIT_VERSION
+
+ARG APP_ENV
+ENV APP_ENV $APP_ENV
+
+RUN apt-get update
+RUN apt-get install -y zip libzip-dev \
+	unzip \
+	vim \
+	sudo \
```
Comment (djodjo3) [inline Dockerfile.prod:19]: Is this part _really_ necessary (though it's used later in the dockerfile) ? 🤔

File: docker/apache2/prestonbot.conf [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        AllowOverride None
+        Order Allow,Deny
+        Allow from All
+
+ 
+        <IfModule mod_rewrite.c>
+            Options -MultiViews
+            RewriteEngine On
+            RewriteCond %{REQUEST_FILENAME} !-f
+	 #  RewriteRule ^(.*)$ app.php [QSA,L]
```
Comment (djodjo3) [inline docker/apache2/prestonbot.conf:15]: Maybe cleaning the commented lines 😛

**Review by djodjo3 — APPROVED** [review]
Some few comments on the side, but a great work on the whole 😉

File: docker/apache2/prestonbot.conf (hunk 3 above)
Comment (JMorinPS) [reply docker/apache2/prestonbot.conf:15]:
```suggestion
```

File: Dockerfile.prod (hunk 2 above)
Comment (JMorinPS) [reply Dockerfile.prod]: zip and unzip packages are necessary to speed up Composer steps, otherwise I removed unecessary ones

**Review by jolelievre — APPROVED** [review]

**Pull request merged**